mods/ra/rules/husks.yaml

```diff
@@ -440,3 +440,25 @@ TC05.Husk:
 		Dimensions: 4,3
 	MapEditorData:
 		ExcludeTilesets: DESERT, INTERIOR
+
+MH60.Husk: 
+	Inherits: ^HelicopterHusk
+	Tooltip:
+		Name: Black Hawk
+	Aircraft:
+		TurnSpeed: 4
+		Speed: 112
+	WithIdleOverlay:
+		Sequence: rotor
+	SmokeTrailWhenDamaged:
+		Offset: -427,0,0
+		MinDamage: Undamaged
+	RevealsShroud:
+		Range: 10c0
+		Type: GroundPosition
+		RevealGeneratedShroud: False
+	RevealsShroud@GAPGEN:
+		Range: 8c0
+		Type: GroundPosition
+	RenderSprites:
+		Image: mh60
```

Comment:
This now needs to be `MH60`, too! (See travis error logs.)

Reply:
Actually, now that I think of it: It is probably better to change `MH60` back to `mh60` in the sequence yamls. Apparently we have all names upper case in the rules, but lower case in the sequences.

Reply:
I see, made it in sequences be lower case